### ISSUE TITLE
Invalid domain recaptcha crashes getToken()

### ISSUE THREAD

**Issue by Allcharles** [armenstepanyan/ng-recaptcha3#16]
If the domain for a website is not valid for the recaptcha seed, it will throw an error while calling `getToken()`.

## Steps to Re-Produce
Valid domains for recaptcha token
![image](https://user-images.githubusercontent.com/3955116/112786650-149b0880-909a-11eb-8530-eff5807a85ac.png)

If I remove one of the domains in the above image, and run the following code:

```typescript
class FormComponent {
  public constructor(recaptcha: NgRecaptcha3Service) { }

  public onSubmit() {
    const seed = "recaptcha seed";
    const status = await this.recaptcha.init(seed);
 
    if (status === "error") {
      return console.error("Failed to initialise recaptcha");
    }
  
    console.log(await this.recaptcha.getToken());
  }
}
```

I will get the following error:
![image](https://user-images.githubusercontent.com/3955116/112787138-406abe00-909b-11eb-96fc-cb37eb9dcf05.png)

Currently, my workaround is the following:

```typescript
class FormComponent {
  ...

  public onSubmit() {
    ...
    try {
      console.log(await this.recaptcha.getToken());
    } catch(err) {
      console.error("Recaptcha failed to retrieve token");
    }
  }
}
```

## Proposed Solutions

1. Detect invalid domain during `init()` if possible?
2. Gracefully handle error thrown in `getToken()`
3. Document `getToken()` can throw errors, and detail the type of errors

**Comment by Allcharles** [comment]
If you would like, I am happy to create a PR to solve the issue

**Comment by armenstepanyan** [comment]
I've wrapped `execute`  function with `try/catch` block, and in case of error `getToken` will return `Promise.reject with` error message

**Comment by armenstepanyan** [comment]
Fixed in version `1.3.2`
